include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2251,7 +2251,7 @@ namespace internal
                                temp1,
                                values_quad + comp * n_q_points,
                                gradients_quad + comp * dim * n_q_points,
-                               scratch_data + 2 * n_components * dofs_per_face,
+                               scratch_data + 2 * dofs_per_face,
```

Comment:
This is just a refactoring step, isn't it?

Reply:
Yes, this was forgotten in #10904.